libs/sdk-ui-ext/src/internal/dashboardEmbedding/index.ts

```diff
@@ -2,3 +2,4 @@
 export { useDashboard, IUseDashboardConfig } from "./useDashboard";
 export { useDashboardAlerts, IUseDashboardAlertsConfig } from "./useDashboardAlerts";
 export { KpiView, IKpiViewProps } from "./KpiView";
+export { DashboardView, IDashboardViewProps } from "./DashboardView";
```

Comment:
please make sure these have API maturity annotations

(ext does not have api-extractor enabled atm due to AE deficiencies)

Reply:
added